spec/datatype_vs.txt

```diff
@@ -297,20 +297,20 @@ Body location is a flexible structure that allows the location to be determined
 * Code only: The code should include (precoordinate) laterality and/orientation to the degree necessary to completely specify the body location.
 * Code plus laterality and/or orientation: The basic code augmented by codes specifying the body side and/or anatomical orientation.
 * Relation to landmark: The location relative to a landmark is specified by:
-    - Establishing the location and type of landmark using a body site code and optional laterality/orientation, and 
-	- Specifying the direction and distance from the landmark to the body location. 
+  * Establishing the location and type of landmark using a body site code and optional laterality/orientation, and 
+  * Specifying the direction and distance from the landmark to the body location. 
 
 Note that BodyLocation is a data type (a reusable structure), not a stand-alone entity. The concept is similar to how a postal address can apply to a person, location, or organization. This contrasts with FHIR's stand-alone BodySite (aka BodyStructure in r4) which 'is not ... intended for describing the type of anatomical location but rather a specific body site on a specific patient' (FHIR 3.5)."
 Includes codes descending from SCT#123037004  "Body Structure"
 
 ValueSet:       LateralityVS
 Description:    "Terms that specify the side of the body. The laterality value set is part of BodyLocation, a flexible structure that allows the location to be determined by a single code, or a code plus laterality and/or orientation. The body location can also be specified in relation to one or more body landmarks. SNOMED CT is used in all cases. 
 
-+ Code only: The code should include (precoordinate) laterality and/orientation to the degree necessary to completely specify the body location.
-+ Code plus laterality and/or orientation: The basic code augmented by codes specifying the body side and/or anatomical orientation.
-+ Relation to landmark: The location relative to a landmark is specified by:
-    - Establishing the location and type of landmark using a body site code and optional laterality/orientation, and 
-	- Specifying the direction and distance from the landmark to the body location. 
+* Code only: The code should include (precoordinate) laterality and/orientation to the degree necessary to completely specify the body location.
```

Comment:
sub-bullets do not show with this change, but will be addressed in my next PR which addresses the sub-bullet content bug in https://standardhealthrecord.atlassian.net/browse/MCODE-43.